bats/tests/helpers/vm.bash

```diff
@@ -62,7 +62,7 @@ EOF
 start_kubernetes() {
     start_container_engine \
         --kubernetes-enabled \
-        --kubernetes-version "$RD_KUBERNETES_PREV_VERSION"
+        --kubernetes-version "$RD_KUBERNETES_PREV_VERSION" "$@"
```

Comment:
Since we use one line per option I would prefer to move `"$@"` to a new line:

```suggestion
        --kubernetes-version "$RD_KUBERNETES_PREV_VERSION" \
        "$@"
```